From d3ce16073818c1af2895578801267bf5408205bd Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Wed, 22 Apr 2020 11:32:49 +0700
Subject: [PATCH 501/504] ARM: dts: r8a7744: rcar-du: Add list of VSP phandles

This patch adds phandles which identify the VSP instance that serves the
DU channel.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 47fbb6b..3456152 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -1713,6 +1713,8 @@
 			clock-names = "du.0", "du.1", "lvds.0";
 			status = "disabled";
 
+			vsps = <&vspd0 &vspd1>;
+
 			ports {
 				#address-cells = <1>;
 				#size-cells = <0>;
-- 
2.7.4

